packages/cli/src/commands/type-check.js

```diff
@@ -11,8 +11,8 @@ import { getPaths } from 'src/lib'
 import c from 'src/lib/colors'
 import { generatePrismaClient } from 'src/lib/generatePrismaClient'
 
-export const command = 'typecheck [sides..]'
-export const aliases = ['tsc', 'tc', 'type-check']
+export const command = 'type-check [sides..]'
+export const aliases = ['tsc', 'tc', 'typeCheck', 'typecheck']
```

Comment:
Adding all these aliases feels like unhelpful overhead to me. I think we should have _one_ alias for each and leave it at that. So `type-check` would be `tc` aliased. Boom.

The only exception would be if we make a change to `dataMigrate`, because it's used in the deploy command. It's currently wrapped, so we can update easily. But because it has stand-alone legacy, I'd prefer to 1) give deprecation warning for one release cycle using an alias and then 2) deprecate fully by v1-rc

Reply:
I dunno DP, I think its the otherway. There's nothing more infuriating that you typed `typecheck` instead of `typeCheck` - and it totally isn't obvious why its not working. 

- tsc is here because that's the normal typescript way of doing it, and I think it makes sense to leave that. 
- tc: shorthand
- typecheck - we could remove for sure. What I want to be is consistent though, so if we remove typecheck we also shouldn't have datamigrate (as per your original suggestion)
- typeCheck - this one I think is worth leaving, because its kind of sensible that we support both `type-check` and `typeCheck`.  

As usual, this isn't a hill I'm going to die on 😄 - but I did want to lay out all the facts.

Reply:
Turns out I _was_ right. If you use snake case i.e. `type-check` to define the command, yargs will automatically add the alias for typeCheck.